src/Tribe/Service_Providers/Processes.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * Class Tribe__Service_Providers__Processes
+ *
+ * @since TBD
+ *
+ * Handles the registration and creation of our async process handlers.
+ */
+class Tribe__Service_Providers__Processes extends tad_DI52_ServiceProvider {
+
+	/**
+	 * Binds and sets up implementations.
+	 */
+	public function register() {
+		if ( ! ( defined( 'DOING_AJAX' ) && DOING_AJAX && ! empty( $_REQUEST['action'] ) ) ) {
```

Comment:
Use doing Ajax from `Tribe__Main`
I would use also `tribe_get_request_var` to allow the check.

Reply:
the way the async request handling works is by sending `POST` requests to the `wp-admin-ajax` endpoint.
I feel now, reading it, the check should be **stricter**, on the `POST` array only, in place of **broader** (as `tribre_get_request_var` would allow) to take only the real supported case into account.